OneCommunityGlobal/HighestGoodNetworkApp
# add format action and run format

# Description
This PR is for adding a new action in Checks

## Related PRS (if any):
include some changes in #1003 

## Main changes explained:
`.github/workflows/node.js.yml`:  add `npm run format` action here for GitHub checks
`package.json`: change the order when run `npm run format:fix`
other files are directed changed by run `npm run format:fix`

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as any user account
4. test if any function work as normal, which means nothing break because of this PR.

## Screenshots or videos of changes:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/5071040/711f6df3-50a9-4a27-a9ff-144ba1226c2a)

## Note:
NA
